src/CommonITILValidation.php

```diff
@@ -1452,17 +1569,171 @@ public static function rawSearchOptionsToAdd()
             'id'                 => '59',
             'table'              => 'glpi_users',
             'field'              => 'name',
-            'linkfield'          => 'users_id_validate',
+            'linkfield'          => 'items_id_target',
             'name'               => __('Approver'),
             'datatype'           => 'itemlink',
             'right'              => static::$itemtype == 'Ticket' ? ['validate_request', 'validate_incident'] : 'validate',
             'forcegroupby'       => true,
             'massiveaction'      => false,
             'joinparams'         => [
+                'condition'          => [
+                    'REFTABLE.itemtype_target' => User::class,
+                ],
                 'beforejoin'         => [
                     'table'              => static::getTable(),
                     'joinparams'         => [
-                        'jointype'           => 'child'
+                        'jointype'           => 'child',
+                    ]
+                ]
+            ]
+        ];
```

Comment:
:warning: It looks like this search option duplicates the ID 7 a few lines above. In GLPI 9.5, 59 does not exists. This may require  investigation